explorer/websockethandlers.go

```diff
@@ -21,6 +21,26 @@ import (
 // close an already closed connection.
 var ErrWsClosed = "use of closed network connection"
 
+func trimTxInfo(txs []*TxInfo) []*TrimmedTxInfo {
```

Comment:
Please move this function to explorertypes.go.  Generally for functions that just perform simple conversions without any major dependencies, it is preferable to have them in a conversions.go or types.go kind of file.

Reply:
This function is actually no longer used/needed, I'm taking it out